### ISSUE TITLE
Patch(Reddit): Hide recommended communities shelf failed on 2024.34.0

### ISSUE THREAD

**Issue by selfmusing** [inotia00/ReVanced_Extended#2327]
### Type

Error while patching

### Tools used

CLI

### Application

Reddit - v2024.34.0

### Bug description

Patch logs : SEVERE: Executing Hide recommended communities shelf failed:
app.revanced.patcher.patch.PatchException: Failed to resolve CommunityRecommendationSectionFingerprint
        at app.revanced.util.BytecodeUtilsKt.getException(BytecodeUtils.kt:32)
        at app.revanced.util.BytecodeUtilsKt.resultOrThrow(BytecodeUtils.kt:24)
        at app.revanced.patches.reddit.layout.communities.RecommendedCommunitiesPatch.execute(RecommendedCommunitiesPatch.kt:35)
        at app.revanced.patches.reddit.layout.communities.RecommendedCommunitiesPatch.execute(RecommendedCommunitiesPatch.kt:19)
        at app.revanced.patcher.Patcher$execute$1.invokeSuspend$executePatch(Patcher.kt:124)
        at app.revanced.patcher.Patcher$execute$1.invokeSuspend(Patcher.kt:144)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:129)
        at app.revanced.cli.command.MainCommand.run$lambda$30$run(MainCommand.kt:473)
        at app.revanced.cli.command.MainCommand.run(MainCommand.kt:386)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2030)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at app.revanced.cli.main.MainKt.main(Main.kt:7)

### Solution

Need to bump compatibility of "hide recommended communities shelf" patch 

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I didn't use any settings marked as `Experimental Flags`.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Aryan-Chauhan** [comment]
java -jar cli.jar -a reddit.apk -o Reddit -m integrations.apk --options options.json -b patches.jar -e Hide recommended communities shelf

Termux is not recognizing this command.

What is the correct command to exclude a particular patch?

![Screenshot_2024-08-26-08-35-32-75_84d3000e3f4017145260f7618db1d683.jpg](https://github.com/user-attachments/assets/19ab9900-c626-47e0-95f4-0cacee72296a)

**Comment by selfmusing** [comment]
> java -jar cli.jar -a reddit.apk -o Reddit -m integrations.apk --options options.json -b patches.jar -e Hide recommended communities shelf
> 
> Termux is not recognizing this command.
> 
> What is the correct command to exclude a particular patch?
> 
> ![Screenshot_2024-08-26-08-35-32-75_84d3000e3f4017145260f7618db1d683.jpg](https://private-user-images.githubusercontent.com/91815107/361292710-19ab9900-c626-47e0-95f4-0cacee72296a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ2ODc5NTUsIm5iZiI6MTcyNDY4NzY1NSwicGF0aCI6Ii85MTgxNTEwNy8zNjEyOTI3MTAtMTlhYjk5MDAtYzYyNi00N2UwLTk1ZjQtMGNhY2VlNzIyOTZhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI2VDE1NTQxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0MWZjMjRjZmIwYmE3ZjYwZDFkOTI3YmZhNTEyM2NhNmQwOThmYjNiOGU5MzIxZjBmMjVhYmMzY2FjNjhlMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cvPcEAODn5vMcyn8ELGY-MFwgknQfcsnX-fYTCErOrc)

```
java -jar cli.jar -o Reddit -b patches.jar -e "Hide recommended communities shelf"  -m integrations.apk -a reddit.apk
```

**Comment by inotia00** [comment]
fixed in [revanced-patches-arsclib-2.179.1](https://github.com/inotia00/revanced-patches-arsclib/releases/tag/v2.179.1)
